Source/Frontend/UI/Colors.cs

```diff
@@ -0,0 +1,180 @@
+namespace RTCV.UI
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Drawing;
+    using System.Linq;
+    using System.Windows.Forms;
+    using RTCV.CorruptCore;
+
+    public static class Colors
+    {
+        //public static Color generalColor = Color.FromArgb(60, 45, 70);
+        public static Color GeneralColor = Color.LightSteelBlue;
+
+        public static Color Light1Color;
```

Comment:
Most of these colors aren't referenced anywhere, but I'm leaving them as public because I believe the intention is for them to be easily consumed as a "theme" by other components.

Reply:
So actually, you may be able to find some of these colors within strings within the form. We use the "tag" property of Winforms to drive the color. You can here https://github.com/ircluzar/RTCV/pull/87/files/02c15d9f3a98ce161138f34d7d6da7911e2b3c7c#diff-cf810005f0f130b8b88f9fcd8fbc4dc4R85 that a dictionary is built based on the colors. This is, as you determined, our loose theme system where all the controls will adjust their color with automatic lightness variations.